post/clang_tidy_review/clang_tidy_review/review.py

```diff
@@ -141,6 +147,11 @@ def main():
                 check=True,
             )
 
+    if args.install_commands:
+        install_commands = args.install_commands
```

Comment:
Passing quoted stuff through is a right pain, so I think we need to handle them here:

```suggestion
    install_commands = strip_enclosing_quotes(args.install_commands)
    
    if install_commands:
```